### PR TITLE
Translate to-ruby-from-python (zh_cn)

### PR DESCRIPTION
**Pull request by Huibean**
Related #1074 Translation in of from Python to Ruby(zh_cn)

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie @AndyHelix
@Zane5 @VirgilMing @boyishwei @dandananddada @jundechen @mohongshu

Please review.

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 1]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+Ruby 与 Python 类似的地方...
+
+* 同样有互动的提示操作环境（叫做 `irb`）。
+* 你同样可以使用指令来读取文件（Python 使用 `pydoc` 指令，而 Ruby 使用 `ri`指令）。
+* 每行程序同样不需要特别的结尾符号（除了一般的换行）。
+* 字符串可以像 Python 的三引号字符串一样的跨越多行。
+* 中括号用来表示串列，大括号用来表示字典结构（在 Ruby 称为 “hashes”）。
+* 阵列操作起来差不多（阵列相加会产生一个更长的阵列，但像 `a3 = [ a1, a2 ]` 这样的操作会回调一个二维阵列）。
+* 对象是健壮类型而且也都是动态类型。
+    所有东西都是对象，变量只是指向某个对象的参考指标。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Miss a asterisk

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 2]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+### 相似点
+
+Ruby 与 Python 类似的地方...
+
+* 同样有互动的提示操作环境（叫做 `irb`）。
+* 你同样可以使用指令来读取文件（Python 使用 `pydoc` 指令，而 Ruby 使用 `ri`指令）。
+* 每行程序同样不需要特别的结尾符号（除了一般的换行）。
+* 字符串可以像 Python 的三引号字符串一样的跨越多行。
+* 中括号用来表示串列，大括号用来表示字典结构（在 Ruby 称为 “hashes”）。
+* 阵列操作起来差不多（阵列相加会产生一个更长的阵列，但像 `a3 = [ a1, a2 ]` 这样的操作会回调一个二维阵列）。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Maybe replace '阵列‘ with '数组' is more suitable？

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 3]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
 
-{% include out-of-date.html %}
+Python是另一种很棒的多功能用途编程语言。从  Python 转换到 Ruby，你会发现需要比学 Python 时多记一些语法。
+
+### 相似点
+
+Ruby 与 Python 类似的地方...
+
+* 同样有互动的提示操作环境（叫做 `irb`）。
+* 你同样可以使用指令来读取文件（Python 使用 `pydoc` 指令，而 Ruby 使用 `ri`指令）。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Maybe translate 'You can read docs on the command line' to '你可以在命令行中阅读文档' is more suitable?

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 4]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+* 每行程序同样不需要特别的结尾符号（除了一般的换行）。
+* 字符串可以像 Python 的三引号字符串一样的跨越多行。
+* 中括号用来表示串列，大括号用来表示字典结构（在 Ruby 称为 “hashes”）。
+* 阵列操作起来差不多（阵列相加会产生一个更长的阵列，但像 `a3 = [ a1, a2 ]` 这样的操作会回调一个二维阵列）。
+* 对象是健壮类型而且也都是动态类型。
+    所有东西都是对象，变量只是指向某个对象的参考指标。
+* 有些关键字虽然看起来不太一样，但在异常处理方面的操作是差不多的。
+* 同样也有嵌入式的文档工具（在 Ruby 中叫做 rdoc）。
+
+### 相异点
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Maybe '不同点' is more common usage?

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 5]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+
+Ruby 与 Python 不同的地方...
+
+* 字符串是可变动的。
+* 你可以建立常量（不想要再改变值的变量）。
+* 有些大小写的使用惯例（例如类别名称是以大写字母开头，一般变量则是使用小写字母开头）。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Maybe we translate 'class names' to '类名' is more suitable than '类别名称'。

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 6]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+* 只有一种串列容器（阵列），而且它是可变动的。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Maybe we translate 'list container (an Array)' to '列表容器（数组）' is more suitable than '串列容器（阵列）'?

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 7]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+* Ruby 的双引号字符串允许使用换码序列 （例如 `\t`） 以及特别的“表达式置换” （expression substitution）语法（可以让你直接在字符串里直接插入 Ruby 预算后的结果，而不用像 `"add " + "strings " + "together"` 的字符串组合）。单引号字符串跟 Python 的 `r"raw strings"`是一样的。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: We need remove space around ’full-width‘(全角) brackets，Maybe 'Ruby 预算后的结果' should be 'Ruby 运算后的结果'

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 8]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+* 没有什么“新式”或”旧式“的类别写法， Ruby 的类别就只有一种写法。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: Maybe translate "classes" to "类" is more suitable than "类别"

File: zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md [hunk 9]
```diff
@@ -4,4 +4,42 @@ title: "从 Python 到 Ruby - Ruby 官方网站"
...
+* 你不能直接存取对象的属性。在 Ruby 里指能通过方法调用。
```
Comment (reyesyang) [inline zh_cn/documentation/ruby-from-other-languages/to-ruby-from-python/index.md]: "指能"　should be "只能"

**Comment by stomar** [comment]
Please use meaningful commit messages (see e.g. new PR title).

**Comment by stomar** [comment]
**Note: This is in conflict with an earlier PR that provides a translation, #1082.**

**Comment by JuanitoFatas** [comment]
Superseded by #1082.

Thank you for your contribution @Huibean, if you have further improvements please start a new branch from current master and open a new pull request. Thanks!

**Pull request closed without merging**